packages/core/src/profile/redux/reducer.js

```diff
@@ -201,6 +211,12 @@ export const contacts = reducerFactory(
   actionTypes,
 );
 
+export const userAttributes = reducerFactory(
```

Comment:
use `createReducerWithResult` instead of `reducerFactory`. This way the result is correctly stored in the `userAttributes` subarea that you created in the beginning.